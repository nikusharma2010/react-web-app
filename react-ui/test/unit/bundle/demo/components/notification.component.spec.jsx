import React from 'react';
import expect from 'expect';
import { create } from "react-test-renderer";
import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import { mount } from 'enzyme';
import Notification from '../../../../../src/bundle/demo/components/notification.component';

/** Check the Application Module exists**/
Enzyme.configure({ adapter: new Adapter() });

describe("Notification Component", () => {
    test("it matches the snapshot", () => {
        const component = create(<Notification message={'Message Check'} />);
        expect(component.toJSON()).toMatchSnapshot();
    });
});

describe('Notification Component', () => {
    let component;
    beforeEach(() => {
        component = mount(< Notification message='Message Check' />);

    });
    it("Check for Notification Message ", function () {
        expect(component.props().message).toEqual('Message Check');
    });
});
