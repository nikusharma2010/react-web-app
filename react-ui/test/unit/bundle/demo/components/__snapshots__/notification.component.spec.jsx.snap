// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Notification Component it matches the snapshot 1`] = `
<div
  className="styles__Alert-sc-18xwbcb-13 lbQqls"
>
  <div
    className="styles__Row-sc-18xwbcb-2 MjgCm"
  >
    <strong>
      Message Check
    </strong>
  </div>
</div>
`;
